pages/linux/repquota.md

```diff
@@ -0,0 +1,27 @@
+# repquota
+
+> Display a summary of existing file quotas for a filesystem.
+
+- Report on all existing quotas:
+
+`repquota -all`
+
+- Report all quotas, even if there is no usage:
```

Comment:
So the quotas reported won't be the limits for each user, but actual usages that they make of their quota? Or both?